### PR TITLE
Adds Mamba_Mayhem game

### LINKED ISSUE #4291: [New game]: Mamba_Mayhem
### 🎮 Game Request

# Game Request
I would like to make a fun and interesting retro-style Snake game with high score functionality.

### Point down the features

- Classic snake gameplay with retro graphics and sound effects
- Smooth and responsive controls
- Increasing difficulty as the snake grows longer
- Display of current score during the game
- High score tracking and display after each game
- Option to reset high scores

## Game points
- Points are earned by eating food items that appear on the screen
- Each food item increases the snake's length and adds to the player's score
- The game ends when the snake collides with itself or the walls

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by Lelin07**
## PR Description 📜

This PR adds a new Mamba_Mayhem game to the repository, which is a classic snake game.

## Fix #4291

Git clone was not working (couldn't solve issue even after increasing HTTP buffer size or shallow cloning) so I have done it with the help of file upload.

<hr>

## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generate no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Screenshots 📷 

<div style="display: flex; justify-content: space-between;">
  Screenshot 1
  Screenshot 2
</div>

---

<br>

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @Lelin07 ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Lelin07,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Lelin07,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Lelin07,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Review by kunjgit — CHANGES_REQUESTED** [review]
Hello @Lelin07,
You need to make the following changes:
  - Please modify the main README.md file.
  - Please add an image to the assets/images directory.

Hoping that you will make those changes soon 🚀

**Comment by Lelin07** [comment]
@kunjgit Kindly review the commits made and merge the PR. Thanks

File: Games/Mamba_Mayhem/assets/images/Mamba_Mayhem.png
Comment (ishita-43) [inline Games/Mamba_Mayhem/assets/images/Mamba_Mayhem.png:1]: ADD THE SAME IMAGE IN ASSEST FODER OF THE GAMEZONE REPOSITORY IN IMAGES SECTION AND ALSO UPDATE THE MAIN README.MD FIILE AND ENTER YOUR GAME THERE FOR REFERENCE SEE OTHER PR MERGED AND THEIR FOLDERS 
hoping your will make changes soon!!

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: Games/Mamba_Mayhem/assets/images/Mamba_Mayhem.png
Comment (Lelin07) [reply Games/Mamba_Mayhem/assets/images/Mamba_Mayhem.png:1]: @ishita-43 Did all the changes. Kindly review all the changes

**Comment by ishita-43** [comment]
Resolve the merge conflicts!!

File: README.md [hunk 1]
```diff
@@ -328,6 +328,7 @@ This repository also provides one such platforms where contributers come over an
 | [Brick Buster Game](https://github.com/kunjgit/GameZone/tree/main/Games/Brick Buster) |
 | [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy) 
 |[Penguins Can't Fly](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Penguins_Can't_Fly)|
+|[Mamba_Mayhem](https://github.com/kunjgit/GameZone/tree/main/Games/Mamba_Mayhem)|
```
Comment (ishita-43) [inline README.md:331]: Place your game 10 places above  as it is giving merge conflicts!!

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (Lelin07) [reply README.md]: @ishita-43 Sorry for any frustrations caused. I'm learning how to resolve merge conflicts. Verify the changes

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by kunjgit** [comment]
Thank you  @Lelin07 , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Pull request merged**